libraries/Microsoft.Bot.PublishValidation/Helpers/ConfigurationOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.PublishValidation
+{
+    public class ConfigurationOptions
+    {
+        public bool ForbidSpacesInProjectName { get; set; }
```

Comment:
Default to true

Reply:
The default values were in [Microsoft.Bot.PublishValidation.targets](/libraries/Microsoft.Bot.PublishValidation/build/Microsoft.Bot.PublishValidation.targets) but had to be updated and so we moved them to the ConfigurationOption class as you suggested.